packages/shorebird_cli/lib/src/commands/patch/patch_android_command.dart

```diff
@@ -358,6 +348,16 @@ ${summary.join('\n')}
     return ExitCode.success.code;
   }
 
+  Future<String?> promptForReleaseVersion(List<Release> releases) async {
+    if (releases.isEmpty) return null;
+    final release = logger.chooseOne(
+      'Which release would you like to patch?',
+      choices: releases,
+      display: (release) => release.version,
+    );
+    return release.version;
+  }
```

Comment:
This logic seems like it could/should be shared among all the patch commands

Reply:
Yeah planning to simplify in a subsequent PR